docker/unittest-dedb/create-image.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+set -e
+set -x
+
+if [ $(docker ps | grep dedb | wc -l) -gt 0 ]; then
```

Comment:
It might not hurt to make the regular expression a little more specific to avoid matching unrelated containers. Something like `grep '\sdedb$'` would match only containers with names that exactly match `dedb` (assuming that container names can't contain whitespace).